querybook/webapp/components/Search/SearchSchemaSelect.tsx

```diff
@@ -58,11 +59,12 @@ export const SearchSchemaSelect: React.FC<ISearchSchemaSelectProps> = ({
         <AsyncSelect
             styles={tableReactSelectStyle}
             placeholder={'search schema name'}
-            value={selectedSchemaItems}
             onChange={handleUpdateSearchFilter}
             loadOptions={loadOptions}
             defaultOptions={[]}
-            noOptionsMessage={() => 'No schema found.'}
+            inputValue={searchText}
+            onInputChange={(text) => setSearchText(text)}
```

Comment:
is these two properties and `searchText` needed? seems it is working well today without them, except the metastore id issue.

Reply:
No they are not, I'll revert.